source/edge-server/configure.txt

```diff
@@ -585,6 +585,77 @@ Where the JSON file contains:
      "Item": "priority >= 3"
    }
 
+.. _edge-server-simulate-offline-connectivity:
+
+Simulate Offline Connectivity
+-----------------------------
+
+Edge Server provides a demo mode you can use to simulate the instance losing
+connectivity with Atlas. While the Edge Server instance is in offline demo 
+mode, you can continue to perform CRUD operations with the instance using a
+MongoDB Driver or tool, or an Atlas Device SDK application.
+
+You can see the data change locally with clients connected to the Edge Server
+instance, but do not see the changes reflected in Atlas. Similarly, changes
+that you make in Atlas while the Edge Server instance is offline do not sync
+to the Edge Server and reflect on local clients.
+
+When you bring the Edge Server instance out of offline demo mode, it
+automatically resumes syncing with Atlas. You can see the data from the 
+Edge Server instance reflected in Atlas, and data changed in Atlas becomes
+visible to the instance and connected clients.
+
+Set up Offline Demo Mode
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Use the following ``edgectl`` flag with either the ``init`` command or the
```

Comment:
does it make sense to spell it out as: you can either create a new instance with the demo or add it to an existing instance?

Reply:
I think people are very unlikely to use it with `init` so I wanted to gloss over that - I think the real use case is `config`. But it's a valid flag for `init` so it seemed like I should at least mention that.